framework/src/modules/http_api/helpers/error_handlers.js

```diff
@@ -14,30 +14,32 @@
 
 'use strict';
 
+const { TransactionError } = require('@liskhq/lisk-transactions');
+
 /**
  * Converts array of errors into string
  *
  * @class
  * @memberof helpers
- * @param {array | object} errors
+ * @param {array | object} e
  * @see Parent: {@link helpers}
  * @todo Add description for the params
  */
 // TODO: Move this functionality to logger component
-function convertErrorsToString(errors) {
-	if (Array.isArray(errors) && errors.length > 0) {
-		return errors
-			.filter(e => e instanceof Error)
+function convertErrorsToString(e) {
```

Comment:
prefer to use `errors`, but any reason changed?

Reply:
this is a temporary file that I think should be moved to `logger`. the reason I changed is because it can also refer to a single error or string.